.prettierignore

```diff
@@ -6,3 +6,4 @@
 **/dist
 **/query-codemods/**/__testfixtures__
 pnpm-lock.yaml
+packages/**/tsup.config.bundled*.mjs
```

Comment:
because of this error

<img width="1259" alt="image" src="https://github.com/TanStack/query/assets/61593290/a5123e2d-7898-4c42-9777-8c18150fe2fa">